src/Plugin.php

```diff
@@ -1018,7 +1024,11 @@ public function activate($ID)
                     Plugin::class,
                     3,
                     "setup",
-                    $log_message
+                    sprintf(
+                        __('Plugin %1$s has been activated by %2$s.'),
+                        $this->fields['name'],
+                        User::getNameForLog(Session::getLoginUserID(true))
```

Comment:
This will probably fail with an error if you try to activate the plugin from console command.

Reply:
Seems ok, without any error.

![image](https://user-images.githubusercontent.com/470612/214539408-22f1d085-f828-4b5b-801e-44d52b88b547.png)

![image](https://user-images.githubusercontent.com/470612/214539417-0634d094-67a4-44b3-89dd-3c062e45c531.png)